{{cookiecutter.repo_name}}/{{cookiecutter.readme}}

```diff
@@ -4,17 +4,22 @@
 [![PyPI Version](https://img.shields.io/pypi/v/{{ cookiecutter.distribution_name }})](https://pypi.org/project/{{ cookiecutter.distribution_name }}/)
 ![License](https://img.shields.io/pypi/l/{{ cookiecutter.distribution_name }}?color=blue)
 
-{{ cookiecutter.description }}
+{{ cookiecutter.description }}.
 
-Installation
-------------
+
+## Installation
 
 This plugin can be installed via:
 
     python -m pip install {{ cookiecutter.distribution_name }}
 
-Contributing
-------------
+
+## Usage
+
+<<Add plugin details here>>
```

Comment:
Love the idea of adding a blank section to encourage folks to add more details. Great! 😁